fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/expression/DebugExpNodeVisitor.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query.expression;
+
+import java.time.Instant;
+
+import com.ibm.fhir.database.utils.query.Select;
+
+
+/**
+ * Debug version of the {@link StringExpNodeVisitor} which is used to
+ * render the string with the bind variable values written in-place,
+ * which is handy for debug.
+ */
+public class DebugExpNodeVisitor extends StringExpNodeVisitor {
+    private final static String NULL = "NULL";
```

Comment:
SqlConstants?

Reply:
Constants to be revisited after removal of old query builder.